### PR TITLE
feat(command-functions): update fauna crud template

### PR DESCRIPTION
**Pull request by fireridlle**
Update faunadb-js package version.
Rewrite FQL to modern style

### CODE REVIEW

File: src/functions-templates/js/fauna-crud/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "fauna-crud",
-  "version": "1.0.0",
+  "version": "2.0.0",
```
Comment (JGAntunes) [inline src/functions-templates/js/fauna-crud/package.json:3]: I don't believe bumping this version is required, as this is used for template purposes.

```suggestion
  "version": "1.0.0",
```

**Review by JGAntunes — CHANGES_REQUESTED** [review]
Thanks for your contribution @fireridlle! 🙏 Just a small note and I think we're good to go 👍

**Comment by fireridlle** [comment]
@JGAntunes thank you. updated

**Comment by JGAntunes** [comment]
Seems like our formatter is complaining - https://github.com/netlify/cli/pull/1853/checks?check_run_id=1857985371 - could you possibly fix these and push again? `npm run format:fix:prettier` should take care of it.

**Review by JGAntunes — APPROVED** [review]

**Comment by JGAntunes** [comment]
Released under `v3.7.0`. Thanks @fireridlle 👍  - https://www.npmjs.com/package/netlify-cli/v/3.7.0

**Pull request merged**